db/schema_tables.cc

```diff
@@ -1790,7 +1790,7 @@ template<typename V>
 static std::vector<V> get_list(const query::result_set_row& row, const sstring& name);
 
 // Create types for a given keyspace. This takes care of topologically sorting user defined types.
-template <typename T> static std::vector<user_type> create_types(keyspace_metadata& ks, T&& range) {
+template <typename T> static future<std::vector<user_type>> create_types(keyspace_metadata& ks, T&& range) {
```

Comment:
Note (unrelated): such functions are poor candidates for templates, they cause code bloat for no reason. They should accept some concrete type for the range, and the caller should adapt.

Reply:
+1